docs/source/scala/quickstart.md

```diff
@@ -0,0 +1,159 @@
+# Quick Start Guide
+In this section, we will see how we can work with the Raphtory Scala APIs to create and analyse graphs. The foundations that we have build in the introductory chapters are as much applicable to this section therefore we won't be revisiting them here again!
+
+## Installing Dependencies
+To work with a Scala project you would need Scala, Java and SBT as development dependencies. The dependencies that we have tested the project discussed in this section to work with Raphtory are Scala version `2.13.7`, Java version `11.0.11.hs-adpt` and SBT version `1.8.0`.
```

Comment:
`To work with a Scala project you need Scala 2.13, Java 11 and SBT ...` i.e., no `would` and mention version requirements